File: python/days-alive/days-alive.py
from datetime import date, datetime
date_format = "%m/%d/%Y"
today =  date.today()
today = today.strftime(date_format)
your_birthday = input("Your birthday (Example: 07/11/1997): ")
a = datetime.strptime(your_birthday, date_format)
b = datetime.strptime(today, date_format)
delta = b - a
print(f"As of today you are {delta.days} days old.")

File: php/days-alive/days-alive.php
<?php
$date_format = 'm/d/Y';
$today = new DateTime();
$today = $today->format($date_format);
$your_birthday = readline("Your birthday (Example: 07/11/1997): ");
$a = DateTime::createFromFormat($date_format, $your_birthday);
$b = DateTime::createFromFormat($date_format, $today);
$delta = $b->diff($a);
echo "As of today you are " . $delta->days . " days old.";
